gauntlet/packages/gauntlet-solana-contracts/networks/.env.local

```diff
@@ -1 +1,9 @@
-NODE_URL=http://127.0.0.1:8899
\ No newline at end of file
+NODE_URL=http://127.0.0.1:8899
+
+PROGRAM_ID_ACCESS_CONTROLLER=DzzjdPWNfwHZmzPVxnmqkkMJraYQQRCpgFZajqkqmU6G
+PROGRAM_ID_OCR2=HW3ipKzeeduJq6f1NqRCw4doknMeWkfrM4WxobtG3o5v
+PROGRAM_ID_DEVIATION_FLAGGING_VALIDATOR=EnUkr8ruCbCe1EidYghGMYanjz6ifXAxphWZmLMri9Zg
```

Comment:
Which program IDs are these?
Why not use the ones shared for the `localnet` setup at `./contracts/artifacts/localnet/*-keypair.json`?

Reply:
Those were randomly generated, just removed them